### PR TITLE
Updated client endpoints section to include mobile clients

### PR DESCRIPTION
**Pull request by FilippSe**
Updated client endpoints section

### CODE REVIEW

**Comment by scanum** [comment]
@JohanFreelancer9 Copy editing is needed for this PR. Thanks.

File: Teams/direct-routing-plan-media-bypass.md [hunk 1]
```diff
@@ -362,9 +362,9 @@ For information about how to configure two trunks on the same SBC, see the docum
 
 ## Client endpoints supported with media bypass
 
-Media bypass is supported with all Teams Desktop clients and Teams Phone Devices. 
+Media bypass is supported with all standalone Teams Desktop clients, Android and iOS clients and Teams Phone Devices.
```
Comment (JohanFreelancer9) [inline Teams/direct-routing-plan-media-bypass.md:365]:
```suggestion
Media bypass is supported with all standalone Teams Desktop clients, Android and iOS clients, and Teams Phone Devices. 
```

File: Teams/direct-routing-plan-media-bypass.md [hunk 2]
```diff
@@ -362,9 +362,9 @@ For information about how to configure two trunks on the same SBC, see the docum
 
 ## Client endpoints supported with media bypass
 
-Media bypass is supported with all Teams Desktop clients and Teams Phone Devices. 
+Media bypass is supported with all standalone Teams Desktop clients, Android and iOS clients and Teams Phone Devices. 
 
-For all other endpoints that do not support media bypass, we will covert the call to non-bypass even if it started as a bypass call. This happens automatically and does not require any actions from the administrator. This includes Skype for Business 3PIP Phones, and Teams Web Clients that support Direct Routing calling (New Microsoft Edge based on Chromium, Google Chrome, Mozilla Firefox). 
+For all other endpoints that do not support media bypass, we will convert the call to non-bypass even if it started as a bypass call. This happens automatically and does not require any actions from the administrator. This includes Skype for Business 3PIP Phones, and Teams Web Clients that support Direct Routing calling (WebRTC based clients running on Microsoft Edge, Google Chrome, Mozilla Firefox).
```
Comment (JohanFreelancer9) [inline Teams/direct-routing-plan-media-bypass.md:367]:
```suggestion
For all other endpoints that do not support media bypass, we will convert the call to non-bypass even if it started as a bypass call. This happens automatically and does not require any actions from the administrator. This includes Skype for Business 3PIP Phones, and Teams Web Clients that support Direct Routing calling (WebRTC based clients running on Microsoft Edge, Google Chrome, and Mozilla Firefox). 
```

File: Teams/direct-routing-plan-media-bypass.md (hunk 1 above)
Comment (JohanFreelancer9) [inline Teams/direct-routing-plan-media-bypass.md:365]:
```suggestion
Media bypass is supported with all standalone Teams Desktop clients, Android and iOS clients, and Teams Phone Devices. 
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review - minor edits.

**Comment by scanum** [comment]
@CarolynRowe PR has been copyedited and is ready for final review, could you please check and merge? Thanks!

**Pull request merged**